app/code/Magebit/CustomEmailText/etc/email_templates.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
```

Comment:
Nebija prasībās, tāpēc viss ok, bet, ja tu būtu taisījis šo, izmantojot custom theme nevis custom moduli, būtu varējis šo failu izlaist un vienkārši aizstāt visus HTML failus, kā to darīji iepriekšējā uzdevumā ar parastajiem PHTML templeitiem.